init.sh

```diff
@@ -2,138 +2,92 @@
 
 ok="Ok, going to the next question."
 
-cat << "EOF"
+read -r -d '' dna_ascii << "EOF"
 
 -. .-.   .-. .-.   .-. .-.   .
   \   \ /   \   \ /   \   \ /
  / \   \   / \   \   / \   \
 ~   `-~ `-`   `-~ `-`   `-~ `-
 
 EOF
+echo "${dna_ascii}"
+
 echo "Welcome! Before adding that folder (Floflis DNA) to your device, we need to configure the install, to make it more lightweight."
 echo ""
-echo "We will ask some questions and configure according to your answers. Please reply by typping a number (according to your choice) and press [ENTER]."
-cat << "EOF"
+echo "We will ask some questions and configure according to your answers. Please reply by typing y/n (according to your choice) and press [ENTER]."
 
--. .-.   .-. .-.   .-. .-.   .
-  \   \ /   \   \ /   \   \ /
- / \   \   / \   \   / \   \
-~   `-~ `-`   `-~ `-`   `-~ `-
+echo "${dna_ascii}"
 
-EOF
 echo "The first question. Do you want to update Floflis DNA from git (GitLab, GitHub) or manually? (P2P updates aren't supported, yet)"
 echo "Installing without git makes it lightweight (ideal for routers and other limited devices), but will require manual effort like you're doing now."
 echo "Update module and .git folder haves approx. 349.2 KB of disk usage."
-echo "How do you want to update Floflis?"
-echo "1 = FROM GIT"
-echo "2 = MANUALLY"
+echo "Do you want to use git to update Floflis? (Alternative is a manual update)"
+echo "[Y/n]"
 read gitormanual
-if [ "$gitormanual" = "1" ]; then
-   echo "$ok"
-fi
-if [ "$gitormanual" = "2" ]; then
-   echo "Removing .git folder..."
-   sudo rm -r .git
-   echo "Removing update module..."
-   sudo rm modules/update
-   echo "$ok"
-fi
-# if [ "$gitormanual" != "1" ] || [ "$gitormanual" != "2" ]; then
-#    echo "PEEEH!"
-#    echo "$gitormanual isn't a valid answer (as we've asked 1 or 2). Restarting..."
-#    sudo ./init.sh && exit
-# fi
-cat << "EOF"
 
--. .-.   .-. .-.   .-. .-.   .
-  \   \ /   \   \ /   \   \ /
- / \   \   / \   \   / \   \
-~   `-~ `-`   `-~ `-`   `-~ `-
+case gitormanual in
+   [nN])
+      echo "Removing .git folder..."
+      sudo rm -r .git
+      echo "Removing update module..."
+      sudo rm modules/update
+      echo "$ok" ;;
+   
+   *)
```

Comment:
BUG: if you type anything except n or N it assumes you wrote y. Will fix soon

Reply:
Issue: https://github.com/Floflis/Floflis-DNA/issues/2